devices/HumanStateProvider/HumanStateProvider.cpp

```diff
@@ -1211,7 +1198,7 @@ bool HumanStateProvider::open(yarp::os::Searchable& config)
         }
     }
     else {
-        yInfo() << "CUSTOM CONSTRAINTS are not defined in xml file.";
+        //yInfo() << "CUSTOM CONSTRAINTS are not defined in xml file.";
```

Comment:
this `else` condition is now empty, can we remove it?

Reply:
Fixed in https://github.com/robotology/human-dynamics-estimation/pull/184/commits/be3f54f2629c6bbb61fc5786ae225d1ec88522c6